.github/workflows/codeql.yml

```
@@ -18,7 +18,7 @@ on:
 
 jobs:
   analyze:
-    name: Analyze (${{ matrix.language }})
+    name: codeql/analyze (${{ matrix.language }})
```

```suggestion
    name: codeql/analyze-${{ matrix.language }}
```
trying to avoid spaces in name (suggestion, feel free to not apply this one)